nuitka/freezer/Standalone.py

```diff
@@ -753,7 +755,9 @@ def _parseDependsExeOutput(filename, result):
     inside = False
     first = False
 
-    for line in open(filename):
```

Comment:
I think for `for line in open(filename)` were better to become something like 

```
with open(filename) as f: 
   for line in f:
```

That preserves the idea of not ever having the file in memory, which is a good idea by default. 

So I think uses of `f.readlines()` added, if you could change those instances? Otherwise looks perfect to me.

Reply:
You are totally right, I did not want to change too many lines idendation. I will update the PR tomorrow (including the rebase).